src/components/Select/Select.jsx

```diff
@@ -29,7 +29,9 @@ export default class Select extends React.Component {
         el &&
         itemsB[ix] &&
         el.id === itemsB[ix].id &&
-        el.title === itemsB[ix].title
+        el.title === itemsB[ix].title &&
+        el.selected === itemsB[ix].selected &&
+        el.disabled === itemsB[ix].disabled
```

Comment:
adding these like this doesn't feel great to me but I wonder if it's really worth comparing both objects key by key... I've love some advice here, if possible :)

Reply:
This feels good to me, to be honest :) This way we can be very specific on what we use to consider two select items equal or not ^_^

If we wanted to make it shorter and more generic, we could go with something like:

```javascript
Object.keys(el).every(attribute => el[attribute] === itemsB[ix][attribute])
```

Reply:
Thanks for the advice @msanroman 😁 I'll leave it explicit for now and we can keep it in mind for the future if we see a need to make it more generic